docs/news.rst

```diff
@@ -635,7 +635,7 @@ Bug fixes
     exception if ``default`` is ``None``.
     (:issue:`6308`, :issue:`6310`)
 
--   :class:`~scrapy.selector.Selector` now uses
+-   :class:`~scrapy.Selector` now uses
```

Comment:
For the record, seeing how you are using `.` as a prefix elsewhere, know you can also use it here after `~`, i.e. `~.Selector`.